src/GSF.CarbonAware/src/Handlers/CarbonAware/CarbonAwareParameters.cs

```diff
@@ -0,0 +1,378 @@
+using CarbonAware.Model;
```

Comment:
Big comment on this file - it's difficult to keep track of which of the Carbon Aware Parameters are stored in this file. I like the separation of one class per file but I'd be open to discussion if there's a compelling reason to do it this way.

Reply:
This change actually belongs to PR #253, which we've marked for GSF review. This could be a point for the story of moving the `CarbonAwareParameters` to `CarbonAware`.